org.webcamoid.Webcamoid.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.webcamoid.Webcamoid
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: webcamoid
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
for a webcam application that's a very broad permission. I'd see `xdg-pictures` as a suitable one.

Reply:
not sure why it would need desktop or download, but it's way better than `host`